src/models/tshirt/Tshirt.cpp

```diff
@@ -177,24 +177,37 @@ namespace tshirt {
     }
 
     /**
-     * Calculate soil field capacity storage, the level at which free drainage stops (i.e., "Sfc").
+     * Calculate soil field capacity storage threshold, the level at which free drainage stops (i.e., "Sfc").
      *
      * @return The calculated soil field capacity storage.
      */
-    double tshirt_model::calc_soil_field_capacity_storage()
+    double tshirt_model::calc_soil_field_capacity_storage_threshold()
     {
         // Calculate the suction head above water table (Hwt)
         double head_above_water_table =
                 model_params.alpha_fc * ((double) STANDARD_ATMOSPHERIC_PRESSURE_PASCALS / WATER_SPECIFIC_WEIGHT);
         // TODO: account for possibility of Hwt being less than 0.5 (though initially, it looks like this will never be the case)
 
-        double z1 = head_above_water_table - 0.5;
-        double z2 = z1 + 2;
+        // distance from the bottom of the soil column to the center of the lowest discretization
+        double trigger_z_m = 0.5;
+
+        // Previous way this was calculated, but which seems to disagree with Fred's implementation ...
+        //double z1 = head_above_water_table - trigger_z_m;
+        //double z2 = z1 + 2;
+        //double lower_lim = model_params.b * pow(z1, ((model_params.b - 1) / model_params.b)) / (model_params.b - 1);
+        //double upper_lim = model_params.b * pow(z2, ((model_params.b - 1) / model_params.b)) / (model_params.b - 1);
+
+        // ... so changing to this to reflect Fred's code
+        double omega = head_above_water_table - trigger_z_m;
+        // Use this several times
+        double one_minus_one_over_b = (1.0 - (1.0 / model_params.b));
+        double lower_lim = pow(omega, one_minus_one_over_b) / one_minus_one_over_b;
```

Comment:
Can you reference the equation number in the document that these various parts are related to/solving?